community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -2048,20 +2078,25 @@ public static void importTool( String... arguments ) throws IOException
         ImportTool.main( arguments, true );
     }
 
-    private String executeImportAndCatchOutput(String... arguments) throws IOException
+    private String executeImportAndCatchOutput( String... arguments ) throws IOException
     {
-        PrintStream originalStream = System.out;
+        PrintStream originalOut = System.out;
```

Comment:
We already have `SuppressOutput` test rule that have possibility to catch all the output